smithy-diff/src/main/java/software/amazon/smithy/diff/evaluators/ChangedEnumTrait.java

```diff
@@ -29,8 +29,9 @@
 import software.amazon.smithy.utils.Pair;
 
 /**
- * Emits a NOTE when a new enum value is added, emits an ERROR when an
- * enum value is removed, and emits an ERROR when an enum name changes.
+ * Emits a NOTE when a new enum value is appended, emits an ERROR when an
+ * enum value is removed, emits an ERROR when an enum name changes, and
+ * emits an ERROR when a new enum value is inserted.
```

Comment:
Throughout this PR, can we be more specific about what inserted means? We mean that an enum value was added but not at the end of the list of enum values. I think clarifying with this little tweak will save countless hours of teams not understanding the error.